### PR TITLE
OSDOCS-12060: updates for CIDR ranges due to masquerade subnet change

### PR DESCRIPTION
**Pull request by JoeAldinger**
Version(s):
4.17

Issue:
https://issues.redhat.com/browse/OSDOCS-12060

Link to docs preview:

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: modules/nw-operator-cr.adoc [hunk 1]
```diff
@@ -327,6 +327,10 @@ If you set this field to `true`, you do not receive the performance benefits of
 |`string`
 |
 The masquerade IPv4 addresses that are used internally to enable host to service traffic. The host is configured with these IP addresses as well as the shared gateway bridge interface. The default value is `169.254.169.0/29`.
+[IMPORTANT]
+====
+For {product-title} 4.17 and later versions, new clusters will use the following default masquerade subnet for IPv4 `169.254.0.0/17`. For upgraded clusters, there is no change to the default masquerade subnet.
```
Comment (jab-rh) [inline modules/nw-operator-cr.adoc:332]:
```suggestion
For {product-title} 4.17 and later versions, clusters use `169.254.0.0/17` as the default masquerade subnet. For upgraded clusters, there is no change to the default masquerade subnet.
```

**Review by jab-rh — COMMENTED** [review]
@JoeAldinger, thanks! Any place that mentions `internalMasqueradeSubnet` is suitable for this treatment, I think.

Only other spot I see is in the API docs so I imagine that would get updated when you update those for 4.17. Thanks, Jason.

**Comment by arghosh93** [comment]
Info wise looks good to me.
4.17 onward, masquerade subnet can also be modified at day 2 by modifying network.operator.openshit.io custom resource.  Should we also mention this in wherever we are going to update about default masquerade subnet?
cc: @kyrtapz

**Comment by JoeAldinger** [comment]
PTAL @huiran0826

**Comment by huiran0826** [comment]
@jechen0648 Can you take a look? Thanks!

**Review by lpettyjo — COMMENTED** [review]
LGTM!

**Pull request merged**